packages/salesforcedx-vscode-core/src/commands/util/postconditionCheckers.ts

```diff
@@ -8,13 +8,23 @@ import {
   CancelResponse,
   ContinueResponse,
   DirFileNameSelection,
+  LocalComponent,
   PostconditionChecker
 } from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import { existsSync } from 'fs';
+import { join } from 'path';
 import { GlobPattern, workspace } from 'vscode';
+import { GlobStrategy } from '.';
 import { nls } from '../../messages';
 import { notificationService } from '../../notifications';
-import { GlobStrategy } from './globStrategies';
+import { getRootWorkspacePath } from '../../util';
+import { MetadataDictionary } from '../../util/metadataDictionary';
+import { PathStrategyFactory } from './sourcePathStrategies';
 
+type OneOrMany = LocalComponent | LocalComponent[];
+type ContinueOrCancel = ContinueResponse<OneOrMany> | CancelResponse;
+
+// TODO: Replace with ComponentOverwritePrompt in subsequent PR
 export class FilePathExistsChecker
```

Comment:
This technically isn't needed anymore. The create template commands can use the new OverwriteComponentPrompt checker, but that can be done in a separate PR to keep this one smaller.

Reply:
Are we splitting it as a separate story or just a follow up PR under the same story ?

Reply:
I have it as a separate story